docs/module/stp.md

```diff
@@ -28,6 +28,7 @@ MSTP/RSTP ports fall back to regular STP upon receiving a plain STP BPDU.
 ## Global Parameters
 
 * **stp.protocol** (one of stp, mstp, rstp or pvrst) -- Global STP flavor to run on supporting nodes, default **stp**
+* **stp.host_edge_port** (bool) -- Configure ports with only hosts connected as 'edge' port_type on platforms that support it, default **True**
```

Comment:
A cosmetic suggestion: I don't like the name of this parameter. How about 'stub_port_type' with the same parameters as the 'port_type', and then you apply it to stub links that have no 'port_type'?

Reply:
Changed, was looking to use ```link.role == 'stub'``` but it doesn't seem to get set in case of VLANs? Code still checking all connected nodes for ```role == 'host'```

Reply:
IIRC the "single router on a segment" logic sets some extra flags, but maybe they are removed before you get to them. Anyway, don't expect this logic to work on VLANs unless you thoroughly tested all possible scenarios (just saying).

Reply:
... oops, you're using _vlan_saved_neighbors, so this would work as that attribute is created in 'post_link_transform' and you're using it in the final 'post_transform' hook.